Vision/proposal.md

```diff
@@ -0,0 +1,62 @@
+# The Mission of the W3C
+
+The W3C leads the community in defining a World Wide Web that is inclusive and respectful of its users, 
+to empower an ethical, equitable, informed, and interconnected society.
+
+## **Introduction**
+### The World Wide Web
+
+The World Wide Web was originally conceived as a tool for sharing information. 
+It has evolved rapidly into a fundamental part of humanity, 
+sparking major social change by providing access to 
+knowledge, education, commerce and shopping, social experiences, civic functions, entertainment, and more.
+
+The Web's amazing success has also led to many unintended consequences that harm society: 
+openness and anonymity have given rise to scams, phishing, and fraud. 
```

Comment:
I assume the 2 next sentences are additional items in the list of harms; having only one item listed right after the `:` and the sentence ending makes it a bit unclear. It also makes it unclear where the list ends ("Technology is not neutral" is no longer a harm). Hinting that this is not an exhaustive list of harms would probably be useful (e.g. by prefixing this as a list of current well-known issues).

Reply:
fixed in #111.